components/runners/ambient-runner/ambient_runner/platform/auth.py

```
@@ -41,6 +41,7 @@
 # time), so updating os.environ mid-run would not reach it without these files.
 _GITHUB_TOKEN_FILE = Path("/tmp/.ambient_github_token")
 _GITLAB_TOKEN_FILE = Path("/tmp/.ambient_gitlab_token")
+_KUBECONFIG_FILE = Path("/tmp/.ambient_kubeconfig")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/.ambient_kubeconfig"

(S108)